ddp-workspace/projects/ddp-basil/src/app/components/app/app.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { NoopScrollStrategy } from '@angular/cdk/overlay';
+import { MatDialog } from '@angular/material/dialog';
+import { TranslateService } from '@ngx-translate/core';
+import { CompositeDisposable, RenewSessionNotifier } from 'ddp-sdk';
+import { SessionWillExpireComponent } from 'toolkit';
+
+@Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html'
+})
+export class AppComponent implements OnInit, OnDestroy {
+    private anchor = new CompositeDisposable();
```

Comment:
#veryminor not sure there is much profit from this CompositeDisposable service since we can user destroySubject + takeUntil
https://medium.com/angular-in-depth/the-best-way-to-unsubscribe-rxjs-observable-in-the-angular-applications-d8f9aa42f6a0